xyz-models/src/main/java/com/here/xyz/models/hub/Tag.java

```diff
@@ -59,6 +60,26 @@ public class Tag implements XyzSerializable {
   @JsonInclude(NON_DEFAULT)
   private boolean system;
 
+  /**
+   * Optional description for the tag. Maximum length is 255 characters.
+   */
+  @JsonView({Public.class, Static.class})
+  private String description;
+
+  /**
+   * Tag author
+   */
+  @JsonView({Public.class, Static.class})
+  @JsonProperty(access = JsonProperty.Access.READ_ONLY)
+  private String author = "";
+
+  /**
+   * Tag creation timestamp
+   */
+  @JsonView({Public.class, Static.class})
+  @JsonProperty(access = JsonProperty.Access.READ_ONLY)
+  private long createdAt = 0;
```

Comment:
Maybe set the model's default value of createdAt to -1 directly. In that case all tag instances that were created from table records, that did not have that value would automatically have that "invalid" value depicting the non-existence of the timestamp.